src/main/java/OutputGenerator.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Locale;
+
+public class OutputGenerator {
+    private static final String LINE_SEPERATOR = "\t____________________________________________________________";
+
+    public static void listOutput(TaskList taskList){
+        System.out.println(LINE_SEPERATOR + "\n\tHere are the tasks in your list:");
+        taskList.printTaskList();
+        System.out.println(LINE_SEPERATOR);
+    }
+
+    public static void doneOutput(TaskList taskList, String input){
+        String[] inputPart = input.trim().split("\\s+",2);
+        int taskIndex = Integer.parseInt(inputPart[1]) - 1;
+        if(taskIndex > taskList.getTaskCount()-1){
+            System.out.println(LINE_SEPERATOR + "\nIndex out of bound.\n" + LINE_SEPERATOR);
+            return;
+        }
+        if(taskList.getTask(taskIndex).getTaskDone()){
+            System.out.println(LINE_SEPERATOR + "\nThe task is done.\n" + LINE_SEPERATOR);
+            return;
+        }
+        taskList.setTaskDone(taskIndex);
+        System.out.println(LINE_SEPERATOR +
+                "\n\tNice! I've marked this task as done:\n\t\t" + taskList.getTask(taskIndex).getTaskInfoFormat()+ "\n" +
+                LINE_SEPERATOR);
+    }
+
+    public static void addTodoOutput(TaskList taskList, String input){
+        String[] inputPart = input.trim().split("\\s+",2);
+        ToDo newTodo = new ToDo(inputPart[1],false);
+        taskList.addTask(newTodo);
+        System.out.println(LINE_SEPERATOR +
+                "\n\tToDo added:\n\t  " +  newTodo.getTaskInfoFormat() +
+                "\n\tNow you have " + taskList.getTaskCount() + " tasks in the list\n" + LINE_SEPERATOR);
+    }
+
+    public static void addDeadlineOutput(TaskList taskList, String input){
+        String[] inputPart = input.trim().split("\\s+",2);
+        String[] nameDoby = inputPart[1].split("/by");
+        String name = nameDoby[0].trim();
```

Comment:
Interesting naming convention, I think "Doby" sounds cute but maybe "DoBy" would be clearer?